helpers/main_helper.py

```diff
@@ -0,0 +1,75 @@
+import random, string, itertools
+
+# English words searched in a grid
+words = open('words.txt', 'r').read().splitlines()
+# All low case letters from English alphabet
+letters = string.ascii_lowercase
+found_words = set()
+
+
+def create_grid(a, b):
+	"""Create grid of letters with a x b dimensions"""
+	if a < 1 or b < 1: raise Exception('Dimensions of grid can\'t be less than 1')
+	global grid_letters
+	grid_letters = [list(map(lambda let: random.choice(letters), range(a))) for _ in itertools.repeat(object, b)]
+
+
+def grid_transpose(b):
+	"""Transpose grid"""
+	global transpose_letters
+	transpose_letters = [row[i] for i in range(len(grid_letters[0])) for row in grid_letters]
+	transpose_letters = [transpose_letters[x:x + b] for x in range(0, len(transpose_letters), b)]
```

Comment:
could you explain what this is doing?

Reply:
I've been working on this project for a day in total and wanted create this diagonals algorithm myself, so here is what I've done: (screenshots attached)
first `for` loop -> all diagonals from left to right(starts with first element) then up
second `for` loop -> all diagonals from right to left(starts with last element) then up
third `for` loop -> all diagonals for from to right(starts with second element) then down
fourth `for` loop -> all diagonals for from to right(starts with second element) then down

<img width="410" alt="diagonals_primary_down" src="https://user-images.githubusercontent.com/50090715/56926655-3e0f1c00-6aa0-11e9-993e-61dfa050d6e1.png">
<img width="409" alt="diagonals_primary_up" src="https://user-images.githubusercontent.com/50090715/56926656-3e0f1c00-6aa0-11e9-9e29-9dd41a37f83b.png">
<img width="409" alt="diagonals_secondary_down" src="https://user-images.githubusercontent.com/50090715/56926657-3e0f1c00-6aa0-11e9-84c0-ed90fef4ae70.png">
<img width="409" alt="diagonals_secondary_up" src="https://user-images.githubusercontent.com/50090715/56926658-3e0f1c00-6aa0-11e9-99b0-dee4ac92e780.png">

Reply:
```[['g', 'k', 'x', 'm'],
 ['e', 'r', 'p', 'j'],
 ['j', 's', 'x', 'c'],
 ['f', 'z', 'j', 'q']] 
       to
[['g', 'e', 'j', 'f'], 
['k', 'r', 's', 'z'], 
['x', 'p', 'x', 'j'], 
['m', 'j', 'c', 'q']]





```

Reply:
converting rows to columns

Reply:
first line does what I described in previous comment